### PR TITLE
fix:  changed <x-fleet-id> and <namespace> to as they were causing render issues in git gui

### PR DESCRIPTION
**Pull request by Michael-McAuley-SSC**
- changed  \<x-fleet-id\> and \<namespace\> to as they were causing render issues in git gui
- templates already updated

### CODE REVIEW

File: solutions/gke/configconnector/gke-cluster-autopilot/setters.yaml [hunk 1]
```diff
@@ -120,7 +120,7 @@ data:
   #
   # network tags to be assigned to GKE nodes
   # The "ids" tag is the recommended approach to ensure that the traffic reaching the GKE clusters is inspected by Cloud IDS.
-  # Furthermore, you would need to have that "ids" tag listed under the mirroredresources of the packet mirroring policy.
+  # Furthermore, you would need to have that "ids" tag listed under the mirrored resources of the packet mirroring policy.
```
Comment (davelanglois-ssc) [inline solutions/gke/configconnector/gke-cluster-autopilot/setters.yaml:123]: please revert this change. mirroredresources is the name of the yaml key

**Review by davelanglois-ssc — APPROVED** [review]

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

**Review by fmichaelobrien — APPROVED** [review]
/lgtm
very nice
